datafusion/physical-plan/src/sorts/sort_preserving_merge.rs

```diff
@@ -746,7 +746,7 @@ mod tests {
 
     // Split the provided record batch into multiple batch_size record batches
     fn split_batch(sorted: &RecordBatch, batch_size: usize) -> Vec<RecordBatch> {
-        let batches = (sorted.num_rows() + batch_size - 1) / batch_size;
```

Comment:
was it a clippy complaint?

Reply:
i did not modify this line manually. but i did run clippy with `--fix`, so yes